source/draw/DrawState.ooc

```diff
@@ -22,8 +20,12 @@ BlendMode: enum {
 }
 
 DrawState: cover {
+	// Used by all
 	target: Image = null
 	inputImage: Image = null
+	// Only used by the write function
+	_originReference := FloatPoint2D new(0.0f, 0.0f) // Start of text
+	// Only used by the draw function
```

Comment:
We have some documentation for the draw stuff already, right? That would be more appropriate there, then.